about/licenses/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: LLNL Software Licensing
+layout: default
+---
+
+## {{ page.title }}
+{: .page-header}
+
+*This page contains information meant for LLNL Employees with instructions for
+how to mark their source code projects. It is not intended to serve as legal
+advice*
+
+### Required Content
+
+All LLNL software must contain the following markings:
+
+#### 1. README file
+
+All projects shall have a `README.md` file at the root of the repository,
+including at least the following information:
+
+- "Getting Started" instructions
+- "Releasability" section, containing the `LLNL-CODE-XXXXXX` number
+
+#### 2. Disclaimer file
+
+At the root of every source code repository shall be the following text in a
+file named `DISCLAIMER`:
```

Comment:
@tgamblin just pointed out that other projects have used `NOTICE` rather than `DISCLAIMER`, including your projects @davidbeckingsale... Any thoughts between the two?

@jbjonesjr, any thought on these two? Does GitHub "officially recognize" one better than the other?

Reply:
GitHub has no preference `at this time`.

Reply:
Titling it NOTICE should be fine.  We just need the legal language disclaiming liability, warranties, etc.